test/CMakeLists.txt

```diff
@@ -11,8 +11,9 @@ add_subdirectory(legacy)
 
 add_subdirectory(extension)
 
+add_subdirectory(image_processing)
```

Comment:
Given that the `image_processing` tests are now in `test/core/image_processing`, shouldn't the path above reflect that ? (Or rather, that entire line should be added to `test/core/CMakeLists.txt`.)

Reply:
>  (Or rather, that entire line should be added to test/core/CMakeLists.txt.)

Yes. I'd stick it at the end, as line 40:

https://github.com/BoostGSoC19/gil-olzhas/blob/4ed7701b47764edf327e7c82f65cf9815026bdc2/test/core/CMakeLists.txt#L30-L39